pkg/applicationserver/observability.go

```diff
@@ -85,6 +86,7 @@ var (
 	evtForwardDataDown = events.Define(
 		"as.down.data.forward", "forward downlink data message",
 		events.WithVisibility(ttnpb.RIGHT_APPLICATION_TRAFFIC_READ),
+		events.WithAuthFromContext(),
```

Comment:
This is NS->AS (line 440 in `applicationserver.go`), not from any user (and soon not authenticated anymore anyway).